database/users.py

```diff
@@ -0,0 +1,23 @@
+from settings import db
+from models.user import User
+import crypt
+
+
+def add_user(user_name, user_password):
```

Comment:
I have actually deleted this file in the branch u've merged into master. All the stuff from here is inside of the user object. I think we can safely delete this file